lib/graphql/mutations.js

```diff
@@ -155,6 +173,39 @@ export const addCreateCollectiveMutation = graphql(createCollectiveQuery, {
   }),
 });
 
+export const addCreateCollectiveMutationV2 = graphql(createCollectiveQueryV2, {
+  options: {
+    context: { apiVersion: '2' },
```

Comment:
An alternative is to import `API_V2_CONTEXT` from https://github.com/opencollective/opencollective-frontend/blob/9490073fb8de9d24e38190a5b921d7e991b6bbf9/lib/graphql/helpers.js#L7

Reply:
Done!